test/ITensorMPS/base/test_dmrg.jl

```diff
@@ -1,6 +1,5 @@
 using ITensors, Test, Random
-
-using ITensors: nsite, set_nsite!
+import ITensors.ITensorMPS: nsite, set_nsite!, site_range
```

Comment:
```suggestion
using ITensors.ITensorMPS: nsite, set_nsite!, site_range
```
Generally I use `using` if I am just using a function, and not overloading it.

Reply:
Great